### DISCUSSION TITLE
Auto on/off discrete nvidia GPU

### DISCUSSION THREAD

**Discussion by noahfraiture** [hyprwm/Hyprland#3442] [Q&A]
Hello, I searched a bit and I didn't find a clear answer, is it possible to auto disable the discrete GPU on laptop to save battery in case of dual gpu with an integrated like intel iris ?
I already found something like that ```env = WLR_DRM_DEVICES,/dev/dri/card1:/dev/dri/card0``` but it appears that if I change the order of that, I won't be able to login. In the current state, my intel iris has the priority, if I give to my nvidia gpu, I loop in the login so I guess it's pointless to search in the direction.
Any idea or advice ?

**Comment by structnull** [comment]
yep in case if you have a rtd3 supported nvidia card it should automatically power off the nvidia card if it is not in use

[rtd3](https://us.download.nvidia.com/XFree86/Linux-x86_64/525.89.02/README/dynamicpowermanagement.html)
